### ISSUE TITLE
Files missing from list

### ISSUE THREAD

**Issue by Unryze** [ladislav-zezula/StormLib#154]
A small issue occurs (again with WC3 maps, god it is annoying), in short, when there are a lot of files in the map and you apply a listfile, some files are not displayed (probably due to the fake files exceeding the actual amount of files), however, if you drag and drop a file that is supposed to be there, MPQ Editor will recognize it as being present.

Map URL: http://wc3maps.com/map/47166/Naruto_RPG_040fix1

Here is a screenshot with the issue:
![Screenshot_3](https://user-images.githubusercontent.com/11413837/56847435-e2039680-68eb-11e9-9a19-749ffbee829f.png)

In short, the file is there, but is simply not displayed.

Also, I know you have been asked about the "read-only" issue, can you perhaps release one version that simply ignores that flag? Or is it possible to somehow do that myself? Just curious.

**Comment by ladislav-zezula** [comment]
I'll check that issue out.

> Also, I know you have been asked about the "read-only" issue, can you perhaps release one version
> that simply ignores that flag? Or is it possible to somehow do that myself? Just curious.

It's more work that you would expect. The reason behind the entire "read-only" thing is protectors. They mess MPQs so hard that's it's beyond repair.

**Comment by Unryze** [comment]
> I'll check that issue out.
> 
> > Also, I know you have been asked about the "read-only" issue, can you perhaps release one version
> > that simply ignores that flag? Or is it possible to somehow do that myself? Just curious.
> 
> It's more work that you would expect. The reason behind the entire "read-only" thing is protectors. They mess MPQs so hard that's it's beyond repair. Everyone who wants to modify a protected map should start with a clean one first.
Aye, I know that myself, explained the same to other people, was just wondering if you could somehow repeat the same magic as you did with 3.5.1.770 which somehow is ignoring all corruptions it can open without read-only :D

Anyways, as usual, thanks for such a quick answer, if you will have any questions will be glad to answer them.

**Comment by ladislav-zezula** [comment]
There is several duplicate entries in the hash table. All of these have BlockIndex set to 0x472 and file name UI%5CFrameDef%5CGlue%5CBattleNetCustomCreatePanel.fdf:
```
HashIndex        HashLow          HashHi             BlockIndex
000003A8	76242F9E	BD0711AA	00000472	
0000126A	51C4C30E	1261CB7B	00000472	
000035D4	909830BE	D0EE885C	00000472
00003EDA	0C9CC14F	CA004090	00000472
```
The fastest solution will be to remove the name "UI%5CFrameDef%5CGlue%5CBattleNetCustomCreatePanel.fdf" from your listfile.
